src/main/java/fruite/store/service/ReadDateService.java

```diff
@@ -0,0 +1,5 @@
+package fruite.store.service;
+
+public interface ReadDateService {
+    String readDate(String fromFileName);

```

Comment:
let's rename it to FileReaderService, do the same for writer class and don't forget to rename your methods inside the class accordingly (readDate -> readFromFile)